crates/nu-cli/src/completions/completion_common.rs

```diff
@@ -185,24 +193,18 @@ pub fn complete_item(
     match components.peek().cloned() {
         Some(c @ Component::Prefix(..)) => {
             // windows only by definition
-            components.next();
```

Comment:
Just to clarify, the reason for removing these `components.next()` calls is because `components` isn't actually being used at all after matching on `components.peek().cloned()`, right?

Reply:
Yes, exactly :)

Reply:
Great, nice catch